src/Uno.UI/UI/Xaml/Window.Skia.cs

```diff
@@ -0,0 +1,197 @@
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Threading;
+using SkiaSharp;
+using Uno;
+using Uno.Disposables;
+using Uno.Extensions;
+using Uno.Logging;
+using Uno.UI;
+using Windows.Foundation;
+using Windows.Foundation.Metadata;
+using Windows.UI.Composition;
+using Windows.UI.Core;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Media;
+
+namespace Windows.UI.Xaml
+{
+	public sealed partial class Window
+	{
+		private static Window _current;
+		private Grid _window;
+		// private PopupRoot _popupRoot;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)